cpp/include/raft_runtime/neighbors/ivf_pq.hpp

```diff
@@ -74,4 +74,12 @@ RAFT_INST_BUILD_EXTEND(uint8_t, uint64_t)
 
 #undef RAFT_INST_BUILD_EXTEND
 
+void save(const handle_t& handle,
```

Comment:
I know it's redundant but we should document these as well since the `raft::runtime` namespaces are likely to be in their own sections of the API docs.